### PR TITLE
[Issue 1051] Production Networking ADR

### PR DESCRIPTION
**Pull request by coilysiren**
## Summary

Relates to #1051

### CODE REVIEW

File: documentation/decisions/infra/0010-production-networking-long-term-state.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- Has a rather long downtime window of about 90 minutes max.
+
+#### 2b. Prepare A New Prod Load Balancer, Wait For DNS Changeover
+
+Nava prepares a new production load balancer URL inside the prod VPC, by deploying a new totally new production application. This application needs a name, lets say prod-beta. prod-beta can be deployed at any time, then MicroHealth is free to update the DNS record on their own time.
+
+- Requires the most prep work from Nava, and the new application would need to be permanently named prod-beta (or whatever we name it). Other name suggestions follow:
+  - prod-alpha (It just can't be called `prod`)
+  - live
+  - prod-0
```
Comment (coilysiren) [inline documentation/decisions/infra/0010-production-networking-long-term-state.md:77]: An idea just occurred to me. We could call this new prod `prod-east`, with an eye to eventually making a backup `prod-west`

?

**Comment by coilysiren** [comment]
From @aplybeah in slack:

> I gave this a read and took some time to think about it. While this is definitely a pain to deal with, it is a significantly smaller pain than kicking this can down the road and having to worry about live production data.
>
>So:
> 1. I see the value in migration not just for our project, but we can share this back with the Nava platform team as well (depending on the state of the template). If we are going to do something this major, now is the best time to do it
> 2. I’m still not sure about what migration method is best. I’m leaning towards having coordination with Microhealth just due to the amount of time spent on the work so far, but I definitely want to hear from those folks first
> 3. In the event that we end up using option 2b…I vote for live as the new name

**Review by aplybeah — APPROVED** [review]

**Review by jamesbursa — COMMENTED** [review]
I agree that doing the migration is needed, staying on `default` will add an ongoing and permanent maintenance burden.

For migration method, my vote would be for 2a. We are early enough in the site's development that some downtime is reasonable imho, and it's overall simpler and easier.

**Comment by coilysiren** [comment]
Given yalls feedback, I'm going to investigate option 2a. Will update the ADR to reflect that.

**Pull request merged**